modules/RTC/RTCUtils.js

```diff
@@ -114,9 +114,14 @@ function getConstraints(um, options) {
         constraints.video = { mandatory: {}, optional: [] };
 
         if (options.cameraDeviceId) {
-            // new style of settings device id (FF only)
-            constraints.video.deviceId = options.cameraDeviceId;
-            // old style
+            // Don't mix new and old style settings for Chromium as this leads
+            // to TypeError in new Chromium versions. @see
+            // https://bugs.chromium.org/p/chromium/issues/detail?id=614716
+            if (!RTCBrowserType.isChrome() && !RTCBrowserType.isOpera()) {
+                // New style of setting device id.
+                constraints.video.deviceId = options.cameraDeviceId;
+            }
+            // Old style.
             constraints.video.optional.push({
```

Comment:
This is a temporary solution, in future we will fully split old and new style constraints when new versions of Chromium and Firefox will have stable support of new constraints format. For more information see https://github.com/jitsi/lib-jitsi-meet/pull/136